src/utils/vc.js

```diff
@@ -327,15 +407,16 @@ export function validateCredentialSchema(credential, schema) {
  * as we support more APIs there are more details associated with each API. Only Dock is supported as of now.
  * @returns {Promise<void>}
  */
-export async function getAndValidateSchemaIfPresent(credential, schemaApi) {
+export async function getAndValidateSchemaIfPresent(credential, schemaApi, context) { // TODO: update to use exanded values
```

Comment:
Can't this function get context from given credential/presentation

Reply:
no because its using expanded credentials, it has no context property